rapidtide/correlate.py

```diff
@@ -785,6 +808,7 @@ def arbcorr(
 def faststcorrelate(
```

Comment:
This function doesn't seem to be used anywhere except for a test (`teststcorr`).